apps/daimo-mobile/src/App.tsx

```diff
@@ -34,10 +43,73 @@ export default function App() {
 }
 
 function AppBody() {
+  const bottomSheetRef = useRef<BottomSheet>(null);
+
+  const snapPoints = useMemo(() => ["50%"], []);
+
+  useEffect(() => {
+    const subscription = RNShake.addListener(() => {
+      bottomSheetRef.current?.expand();
+    });
+
+    return () => {
+      subscription.remove();
+    };
+  }, []);
+
+  const renderBackdrop = useCallback(
+    (props: BottomSheetDefaultBackdropProps) => (
+      <BottomSheetBackdrop
+        {...props}
+        disappearsOnIndex={-1}
+        appearsOnIndex={0}
+        pressBehavior="close"
+      />
+    ),
+    []
+  );
+
+  const noop = useCallback(() => {}, []);
```

Comment:
you can use `useSendDebugLog()` hook after this PR:

https://github.com/daimo-eth/daimo/pull/484